hedera-node/configuration/dev/settings.txt

```diff
@@ -8,7 +8,7 @@ sync.syncProtocolPermitCount,                  2                            # di
                                           #           State           #
                                           #############################
 
-state.saveStatePeriod,                         300                          # differs from mainnet
+state.saveStatePeriod,                         30                          # differs from mainnet
```

Comment:
Since it already differs from mainnet, changed this for dev environments to add more states other than genesis state for reconnect.